std/traits.d

```diff
@@ -1199,12 +1199,14 @@ template ParameterStorageClassTuple(func...)
     static assert(pstc[2] == STC.none);
 }
 
-/*****************
- * Convert string tuple Attribs to ParameterStorageClass bits
+/**
+ * Convert the result of `__traits(getParameterStorageClasses)`
+ * to $(LREF ParameterStorageClass) `enum`s.
+ *
  * Params:
- *      Attribs = string tuple
+ *     Attribs = The return value of `__traits(getParameterStorageClasses)`
  * Returns:
- *      ParameterStorageClass bits
+ *     The bitwise OR of the equivalent $(LREF ParameterStorageClass) `enum`s.
```

Comment:
BTW when you restructure comments, you can remove the leading stars as we agreed on it not being the [DStyle for Phobos](https://dlang.org/dstyle.html#phobos_documentation).
(this is not required, just an FYI)